docs/changelog.rst

```diff
@@ -2,21 +2,45 @@
 Changelog
 *********
 
-2.2.0 (2024-MM-DD)
+2.2.0 (2024-12-16)
 ------------------
 
 New features:
 
-- Add a :doc:`uv plugin</common/craft-parts/reference/plugins/uv_plugin>` for
-  projects that use the uv build system.
+- Add a :ref:`uv plugin<craft_parts_uv_plugin>` for projects that use the `uv
+  <https://docs.astral.sh/uv/>`_ build system.
+- Add a :ref:`Go Use plugin<craft_parts_go_use_plugin>` for setting up a `go
+  workspace` for Go modules.
+- Add an API for :ref:`registering custom source types
+  <how_to_add_a_source_handler>`.
+- Prefer ``craft.git`` as the binary to handle git sources, in environments
+  where it is available.
```

Comment:
```suggestion
- Prefer ``craft.git`` as the binary to handle Git sources, in environments
  where it's available.
```

Reply:
I don't know why I keep avoiding contractions in changelogs.